pkg/cli/admin/prune/images/images_test.go

```diff
@@ -39,6 +39,52 @@ import (
 
 var logLevel = flag.Int("loglevel", 0, "")
 
+func TestPruneWithoutImagesWithRegistryAccess(t *testing.T) {
+	kFake := fakekubernetes.NewSimpleClientset()
+	imageFake := &fakeimagev1client.FakeImageV1{
```

Comment:
Why does image client have its own variable, while apps client and build client are created inline?

Reply:
Because `copy&paste` reasons. Fixed, thanks.